anonip.py

```diff
@@ -56,6 +56,9 @@
     # compatibility for python < 3
     from urlparse import urlparse
 
+if sys.version_info[0] >= 3:
```

Comment:
We can exclude this from coverage, then CI will be green again:

```suggestion
if sys.version_info[0] >= 3:  # pragma: no cover
```